03/README.md

```diff
@@ -0,0 +1,134 @@
+# Lesson 3
+
+## Content
+- Classes in Kotlin
+  - Normal class
+  - Data class
+- Abstract class
+- Interface
+
+## Useful links
+- [Slack channel](https://appport-academy.slack.com/) (invite can be requested via [event link](https://appport.cz/event/appport-academy/) or [Meetup group](https://www.meetup.com/Prague-Mobile-Development-Meetup/events/269765161/))
+- [Github](https://github.com/Concur-Kotlin-Academy/academy2020)
+- Kotlin manual: [link](https://kotlinlang.org/docs/reference/classes.html)
+
+## A little bit of theory
+### Object Oriented Programming (OOP)
+> Object oriented programming allows us to solve complex problems by using objects
+
+More info in [Wikipedia](https://en.wikipedia.org/wiki/Object-oriented_programming)
+
+
+### Classes
+#### What's a class?
+**CLASS** is a blueprint or a set of instructions to build a specific type of object.
+
+![Class](images/class.png)
+
+##### Parts of a class
+1. **Name:** Identifies the class
+2. **Constructor(s):** Function used to initialize the state/properties of newly created objects of the class. Invoked when creating an instance of the class
+3. **Attribute(s)/property(ies):** keep state
+4. **Method(s)/function(s):** Do something
+
+##### Visibility modifiers in Kotlin
+Classes, objects, interfaces, constructors, functions, properties and their setters can have visibility modifiers.
+
+- `private` - visible (can be accessed) from inside the class only.
+- `protected` - visible to the class and its subclass.
+- `internal` - any client inside the module can access them.
+- `public` - visible everywhere.
```

Comment:
Could we have some task that is playing with accessibility modifiers?

Reply:
yes, i'm planning to use different accessibility modifiers in functions and properties while working on `Task 2` of `Working with classes` section 🙂